### PR TITLE
fix(docs,i18n): update FAQ for translations, add intros

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -4,9 +4,13 @@ Read our ["How to Contribute to Open Source Guide"](https://github.com/freeCodeC
 
 ### Can I translate freeCodeCamp's curriculum?
 
-Yes. We will roll out the tools for translating freeCodeCamp's curriculum soon. 
+Yes - Read [this guide](/how-to-translate-files) if you are interested in contributing to translation.
 
-Eventually, we intend to localize freeCodeCamp into several major world languages, starting with: Arabic, Chinese, Hindi, Portuguese, Russian, and Spanish.
+Currently we have the user contributed translations being done in Chinese and Espanol.
```
Comment (nhcarrigan) [inline docs/FAQ.md:9]:
```suggestion
Currently we have the user contributed translations being done in Chinese and Español.
```

File: docs/FAQ.md [hunk 2]
```diff
@@ -4,9 +4,13 @@ Read our ["How to Contribute to Open Source Guide"](https://github.com/freeCodeC
...
+
+At the minute we are building our experience hosting the curriculum in the aforementioned languages. Eventually, we intend to localize freeCodeCamp into several major world languages, like: Arabic, Hindi, Portuguese, Russian and more.
```
Comment (nhcarrigan) [inline docs/FAQ.md:11]:
```suggestion
At this time we are building our experience hosting the curriculum in the aforementioned languages. Eventually, we intend to localize freeCodeCamp into several major world languages, like: Arabic, Hindi, Portuguese, Russian and more.
```

File: docs/index.md [hunk 3]
```diff
@@ -21,19 +21,27 @@ You can help expand and improve the curriculum. You can also update project user
 
 **If you're interested in improving our curriculum, here's [how to contribute to the curriculum](how-to-work-on-coding-challenges.md).**
 
+## Translations
+
+We are localizing freeCodeCamp.org to world languages starting with Chinese and Espanol. We will be expanding the translations to more languages. It's our dream to provide you with resources to learn, no matter the world language you speak.
+
+To help us with this massive effort, we have integrated our open-source code-base & curricula with [Crowdin](https://crowdin.com/).
```
Comment (nhcarrigan) [inline docs/index.md:28]:
```suggestion
To help us with this massive effort, we have integrated our open-source code-base & curriculum with [Crowdin](https://crowdin.com/).
```

One comprehensive curriculum, right? 🙂

**Review by RandellDawson — APPROVED** [review]
LGTM

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**